webapp/src/widgets/Graph/Graph.js

```diff
@@ -1,32 +1,41 @@
 import React from 'react'
 import { Line, defaults } from 'react-chartjs-2'
+import moment from 'moment'
 
 defaults.global.defaultFontSize = '8'
 
-const mockData = {
-	label: ['April 1', 'April 6', 'April 11', 'April 16', 'April 21', 'April 21'],
-	coordinatesData: [3, 10.0, 5.0, 9.0],
-}
+export const Graph = ({ data }) => {
+	const label = []
+	const coordinates = []
 
-const data = {
-	labels: mockData.label,
-	datasets: [
-		{
-			label: '',
-			fill: false,
-			lineTension: 0.1,
-			borderColor: 'rgba(44, 130, 201, 1)',
-			borderWidth: 2,
-			data: mockData.coordinatesData,
-		},
-	],
-}
+	data.length &&
```

Comment:
no need of length check with forEach
you can use `data && data.forEach`

Reply:
updated